### ISSUE TITLE
Do you have any plans to release the code and dataset?

### ISSUE THREAD

**Issue by lcl124252** [yyxssm/PointSSC#2]
Thank you for your great work! Do you have plans to release the code and dataset?

**Comment by yyxssm** [comment]
Yes, thanks for attention to our work! As ICRA2024 will be held in May,  we will release code and dataset before April 30th.

**Comment by 441599828** [comment]
Do you have plans to release the code and dataset?

**Comment by yyxssm** [comment]
Thank you for your attention to our work. The code and dataset have both been open-sourced.
